packages/replay/src/index.ts

```diff
@@ -755,7 +755,7 @@ export class Replay implements Integration {
       }
 
       if (result.category === 'ui.click') {
-        this.triggerUserActivity();
+        void this.triggerUserActivity();
```

Comment:
l: I'm wondering why this method is `async` in the first place. Probably missing something but afaict it's never used in an async context.

Reply:
good point, I'll just make this un-async!